protocol/messenger_backup.go

```diff
@@ -131,37 +131,38 @@ func (m *Messenger) BackupData(ctx context.Context) (uint64, error) {
 
 	cs := append(joinedCs, deletedCs...)
 	for _, c := range cs {
+		_, beingImported := m.importingCommunities[c.IDString()]
+		if !beingImported {
+			settings, err := m.communitiesManager.GetCommunitySettingsByID(c.ID())
+			if err != nil {
+				return 0, err
+			}
```

Comment:
It's important we don't perform backups for communities that are being imported as they otherwise could show up too early in the UI when an import is still ongoing (we don't want to show communities that haven't finished importing yet)